cmplibrary/src/main/java/com/sourcepoint/cmplibrary/util/extensions/ContextExt.kt

```diff
@@ -0,0 +1,30 @@
+package com.sourcepoint.cmplibrary.util.extensions
+
+import android.content.Context
+import android.net.ConnectivityManager
+import android.net.NetworkCapabilities
+import android.os.Build
+
+/**
+ * Extension function that checks whether or not the device is connected to the Internet
+ */
+fun Context.isInternetConnected(): Boolean {
```

Comment:
@bohdan-go-wombat just curious, have you tested the `fun` performance while throttling network or using vpn?

Reply:
like for this extension by the way  👍

Reply:
@Nevazhnovu I haven't tested using VPN or by throttling my network, but I tested on WiFi and by using celular connection